Firebase.config.js

```diff
@@ -0,0 +1,12 @@
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDzawW7qnt7M-SjlPr7_Gg22GAhJNz_0wE",
+  authDomain: "damage-control-ba92f.firebaseapp.com",
+  projectId: "damage-control-ba92f",
+  storageBucket: "damage-control-ba92f.appspot.com",
+  messagingSenderId: "127685083270",
+  appId: "1:127685083270:web:58bf57eb6d5dfaf3bc1107",
+};
```

Comment:
This information should not appear on GitHub, instead use a sample file with the necessary environment variables.

Reply:
Thanks, it's already done.